notification-service/src/constants/SUBSCRIPTIONS.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import { EventSubscription } from '../types';
+import { ETHERSCAN_API_KEY, ETHEREUM_NETWORK } from '../variables';
+import { getEtherscanAPI, getEtherscanURL } from './NETWORKS';
+import { ethers } from 'ethers';
+
+export async function getAbiFromContractAddress(contract: string) {
```

Comment:
I suggest that you move this function to another `etherscan` file, since this one suppose to contain only configuration